modules/seqexec/model/src/main/scala/seqexec/model/NSSubexposure.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model
+
+import cats.Eq
+import cats.implicits._
+import seqexec.model.enum.NodAndShuffleStage
+import seqexec.model.enum.NodAndShuffleStage._
+
+final case class NSSubexposure(
+  totalCycles: Int,
+  cycle:       Int,
+  id:          Int,
```

Comment:
Can we call this `index` maybe, since it's the index into `NsSequence`? It would be good to have doc comments on these parameters.

Reply:
Yeah, now that you mention I'll also use `NsCycle` for the cycle-related fields